common/test/views/support/cleaner/AmpEmbedCleanerTest.scala

```diff
@@ -85,6 +89,40 @@ class AmpEmbedCleanerTest extends FlatSpec with Matchers {
     result.getElementsByTag("amp-vimeo").size should be(1)
   }
 
+
+  /////////////////////////////
+  // Interactive cleaner
+  /////////////////////////////
+
+  "AmpEmbedCleaner" should "create an amp-iframe element if interactive has a valid url and iframe wrapper" in {
+    val interactiveValidUrlPlusiFrameWrapper = s"""<html><body><figure class="element element-interactive interactive"
```

Comment:
you can actually just write html in scala and then use .toString(). This makes it much easier to read and reason about.